lang-guide/chapters/types/basic_types/date.md

```diff
@@ -1,28 +1,18 @@
 # Date
 
-What it is: A value representing a specific point in time using international standard date time descriptors.
+| **_Description:_** | Represents a specific point in time using international standard date time descriptors
```

Comment:
I think this table is missing the top part

Reply:
Thanks! Fixed locally and will resolve in next commit.